pkg/scalers/pulsar_scaler.go

```diff
@@ -0,0 +1,301 @@
+package scalers
+
+import (
+	"context"
+	"encoding/json"
+	"errors"
+	"fmt"
+	"io/ioutil"
+	"net/http"
+	"strconv"
+	"strings"
+
+	"k8s.io/api/autoscaling/v2beta2"
+	"k8s.io/apimachinery/pkg/api/resource"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/labels"
+	"k8s.io/metrics/pkg/apis/external_metrics"
+	logf "sigs.k8s.io/controller-runtime/pkg/log"
+
+	kedautil "github.com/kedacore/keda/v2/pkg/util"
+)
+
+type pulsarScaler struct {
+	metadata pulsarMetadata
+	client   *http.Client
+}
+
+type pulsarMetadata struct {
+	adminURL            string
+	topic               string
+	subscription        string
+	msgBacklogThreshold int64
+
+	// TLS
+	enableTLS bool
+	cert      string
+	key       string
+	ca        string
+
+	statsURL                   string
+	activationTargetQueryValue int64
```

Comment:
I guess that this variable is for the activation feature, please use the same name as the target/threashold value with the prefix `activation`, I think that in this case it would be `activationMsgBacklogThreshold`